ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -719,15 +737,24 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
                   fontSize: '20px', fontWeight: 400}}>
                   Select cohorts, concept sets, and values above to generate a preview table
                 </div>
-                <Button data-test-id='preview-button' disabled={this.disableSave()}
-                        style={{marginTop: '0.5rem', height: '1.8rem', width: '6.5rem'}}
-                        onClick={() => this.getPreviewList()}>
-                  Preview Table
-                </Button>
               </div>
             }
           </div>
         </FadeBox>
+        <div>
+          <div style={styles.footer} />
+          <div style={styles.stickyFooter}>
+            <TooltipTrigger data-test-id='save-tooltip'
+            content='Requires Owner or Writer permission' disabled={this.canWrite}>
+            <Button style={{marginBottom: '2rem'}} data-test-id='save-button'
+            onClick ={() => this.setState({openSaveModal: true})}
+            disabled={this.disableSave() || !this.canWrite}>
+            {this.editing ? !(dataSetTouched && this.canWrite) ? 'Analyze' :
+            'Update And Analyze' : 'Save And Analyze'}
```

Comment:
"And" should be lowercased here

Reply:
Also in the mocks it uses all caps on the button, though that is probably best implemented via CSS if we do that.

Reply:
button change text to all Upper case i can change this just for readability